packages/fs-detectors/src/packageManagers/package-managers.ts

```diff
@@ -0,0 +1,60 @@
+import type { Framework } from '@vercel/frameworks';
+
+export const packageManagers: Array<
+  Omit<Framework, 'description' | 'getOutputDirName' | 'settings'>
+> = [
```

Comment:
Now that we are using the concept of a _framework_ to represent monorepo managers, workspace managers, and package managers, we may want to rework some of this. Maybe at least the naming of things. It shouldn't be `frameworkDetector` but maybe just `entityDetector`, and it shouldn't require (but can still support) all these properties that were added just for frameworks